crane-cloud/frontend
# Ft logs filtering

# Description

Add possible activity logs Filtering on Client side

## Type of change

Please delete options that are not relevant.


## Trello Ticket ID

https://trello.com/c/YnhhmgoO

## How Can This Been Tested?

Pull this branch and check the pages with activity logs.
Note: Data filtering still returns an error. Probably a format issue for the backend team.

## Checklist:


# Screenshots

![image](https://user-images.githubusercontent.com/69305400/222300882-af139878-83db-44bd-bed6-016396c674f1.png)
